h/services/analytics.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+from h.schemas.analytics import Event
+
+
+class AnalyticsService:
+    def __init__(self):
+        self._log = logging.getLogger(__name__)
+
+    def create(self, event: Event):
+        # TODO Enhance this
+        self._log.info(event)
```

Comment:
I mentioned the possibility of using new relic here. I wonder if we could use the new relic JS library directly on the client though.

Just logging does the job now :+1:

Reply:
> I mentioned the possibility of using new relic here. I wonder if we could use the new relic JS library directly on the client though.

We discussed that it probably made sense to funnel any event tracking through our backend, regardless the persistence method.

We can revisit this decision later, if we find it problematic.